src/analyze/analyze.c

```diff
@@ -214,13 +216,92 @@ static int compare_unit_start(const UnitTimes *a, const UnitTimes *b) {
         return CMP(a->activating, b->activating);
 }
 
+static int process_aliases(char *argv[], char *tempdir, char ***ret) {
+        int r, i = 1;
+        char **filename;
+        _cleanup_strv_free_ char **filenames = NULL;
+        struct stat statbuf;
+
+        assert(argv);
+        assert(tempdir);
+        assert(ret);
+
+        STRV_FOREACH(filename, strv_skip(argv, 1)) {
+                _cleanup_free_ char *src = NULL, *dst = NULL, *argname = NULL;
+
+                if (!strchr(*filename, ':')) {
```

Comment:
please don't do half-assed parsing before doing the real parsing with extract_first_word() below.

Just extract both words with extract_first_word() and then handle if the second call hits end of string early